.scalafmt.conf

```
@@ -1,4 +1,4 @@
-version = "3.5.1"
+version = "3.5.2"
```

```suggestion
version = "3.5.3"
```

@valencik can we get this train started again 😆